portfolio/src/main/webapp/script.js

```diff
@@ -113,3 +138,96 @@ function deleteComments() {
     });
   });
 }
+
+let map;
```

Comment:
I would also create a variable called markers to store markers fetched in case we need to use them in the future.